apps/web/src/pages/templates/components/StepNameLabel.tsx

```diff
@@ -0,0 +1,31 @@
+import { Center } from '@mantine/core';
+import { useLocation } from 'react-router-dom';
+
+import { Variant, VariantsFile } from '../../../design-system/icons';
+import { Text, colors } from '../../../design-system';
+import { useStepIndex } from '../hooks/useStepIndex';
+
+export const StepNameLabel = ({ variantsCount = 0 }: { variantsCount?: number }) => {
```

Comment:
this is the smart label component that depending on the context renders the label:
- if root step - render nothing
- if variant - render `Variant {n}`
- if variants list - render `{n} variants`